engine/src/main/resources/application.properties

```diff
@@ -66,16 +66,16 @@ quarkus.log.category."com.redhat.cloud.notifications".level=INFO
 %test.quarkus.http.access-log.category=info
 
 # RBAC configuration used to retrieve email recipients. It is used when an email notification is sent.
-#rbac-s2s/mp-rest/url=http://ci.foo.redhat.com:1337
-rbac-s2s/mp-rest/url=${clowder.endpoints.rbac-service:https://ci.cloud.redhat.com}
-rbac-s2s/mp-rest/connectTimeout=2000
-rbac-s2s/mp-rest/readTimeout=120000
+#quarkus.rest-client.rbac-s2s.url=http://ci.foo.redhat.com:1337
+quarkus.rest-client.rbac-s2s.url=${clowder.endpoints.rbac-service:https://ci.cloud.redhat.com}
+quarkus.rest-client.rbac-s2s.connect-timeout=2000
+quarkus.rest-client.rbac-s2s.read-timeout=120000
 
 # IT User service
 # these entries are needed because of a bug in quarkus: https://github.com/quarkusio/quarkus/issues/8384
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/url=${IT_S2S_MP_REST_URL}/v2
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStore=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE}
-com.redhat.cloud.notifications.recipients.itservice.ITUserService/mp-rest/keyStorePassword=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD}
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".url=${QUARKUS_REST_CLIENT_IT_S2S_URL:FILL_ME}/v2
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store=file:${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE:FILL_ME}
+quarkus.rest-client."com.redhat.cloud.notifications.recipients.itservice.ITUserService".key-store-password=${QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD:FILL_ME}
```

Comment:
Problem solved by adding the default `FILL_ME` which will be used when the env vars are not available.